addon/core/schema.ts

```diff
@@ -55,38 +55,60 @@ export function getRdfaAttrs(node: Element): RdfaAttrs | false {
   };
 
   let hasAnyRdfaAttributes = false;
-  for (const key of Object.keys(rdfaDomAttrs)) {
+  for (const key of [...Object.keys(rdfaDomAttrs), 'about']) {
     const value = node.attributes.getNamedItem(key)?.value;
     if (isSome(value)) {
       attrs[key] = value;
       hasAnyRdfaAttributes = true;
 
-      if (key === 'data-outgoing-props') {
-        const properties = JSON.parse(value) as Property[];
-        attrs['properties'] = properties;
-      }
-      if (key === 'data-incoming-props') {
-        const backlinks = JSON.parse(value) as Backlink[];
-        attrs['backlinks'] = backlinks;
-      }
-      if (key === 'data-rdfa-node-type') {
-        const type = value as unknown as RdfaAttrs['rdfaNodeType'];
-        if (!rdfaNodeTypes.includes(type)) {
-          logger('rdfaNodeType is not a valid type', value, node);
+      switch (key) {
+        case 'data-outgoing-props': {
+          const properties = JSON.parse(value) as Property[];
+          attrs['properties'] = properties;
+          break;
+        }
+        case 'data-incoming-props': {
+          const backlinks = JSON.parse(value) as Backlink[];
+          attrs['backlinks'] = backlinks;
+          break;
+        }
+        case 'data-rdfa-node-type': {
+          const type = value as unknown as RdfaAttrs['rdfaNodeType'];
+          if (!rdfaNodeTypes.includes(type)) {
+            logger('rdfaNodeType is not a valid type', value, node);
+          }
+          if (type === 'resource') {
+            attrs = {
+              ...attrs,
+              rdfaNodeType: type,
+              resource:
+                attrs.resource && typeof attrs.resource === 'string'
+                  ? attrs.resource
+                  : '',
+              properties:
+                attrs.properties && attrs.properties instanceof Array
+                  ? (attrs.properties as Property[])
+                  : [],
+            };
+          }
+          break;
         }
-        if (type === 'resource') {
-          attrs = {
-            ...attrs,
-            rdfaNodeType: type,
-            resource:
-              attrs.resource && typeof attrs.resource === 'string'
-                ? attrs.resource
-                : '',
-            properties:
-              attrs.properties && attrs.properties instanceof Array
-                ? (attrs.properties as Property[])
-                : [],
-          };
+        case 'about': {
+          const about = value as unknown as string;
+          const property = node.attributes.getNamedItem('property')?.value;
```

Comment:
Hmm, so the issue I'm seeing is that if I insert a structure, say a chapter, then the chapter resource has heading and title properties, each of which have the correct backlinks. If I then refresh, the heading is no longer correctly linked to the chapter (either the property on the chapter or the backlink on the heading.

It may well be that there is an issue with the parser that means it's missing these relations (they're in the DOM with an `about` and `property` attribute on the heading.